Source/WebKit/UIProcess/API/Cocoa/WKWebViewConfigurationInternal.h

```diff
@@ -41,7 +44,7 @@ template<> struct WrapperTraits<API::PageConfiguration> {
     API::ObjectStorage<API::PageConfiguration> _pageConfiguration;
 }
 
-@property (nonatomic, readonly) NSString *_applicationNameForDesktopUserAgent;
+@property (nonatomic, readonly, nullable) NSString *_applicationNameForDesktopUserAgent;
```

Comment:
Are the nullability annotations required? I'm a bit surprised this doesn't bleed into the other internal headers when they include WKWebViewConfigurationInternal.h.

Reply:
yes, I get an error when I exclude it